notebookPlots.jmd

```diff
@@ -18,137 +13,166 @@ effects = getODEparams(ps, conc_l)
 g0 = g1[1] + g2[1]
 T = 170
 ```
+
+### Lapatinib
+
 #### Plotting the gradient
+
 ```julia; echo=false
 plotGradient(effects, conc_l, g0, T)
 ```
+
 #### Plotting the time-series simulations compared to the actual data.
```

Comment:
Oh, also, `####` is a paragraph header, where content should come to the right of it, while `###` is a subsubheading, so all content will show below. I bet if you change all the `####` to `###` and all the `###` to `##`, that'll fix the problem.